### ISSUE TITLE
Help enabling addon in osx

### ISSUE THREAD

**Issue by ianm111** [X-Plane/XPlane2Blender#736]
Blender on osx, downloaded addon io….zip, picked zip as addon and its there but tick box greyed out and cannot tick. 
screenshot of error message hopefully attached below;

 <img width="320" alt="Screen Shot 2024-06-16 at 10 24 15" src="https://github.com/X-Plane/XPlane2Blender/assets/153353755/362d3b48-771a-419e-9730-da282903444f">

**Comment by DWmFrancis** [comment]
Ianmm111

Did you remove the old version before attempting to install the new version of xplane2blender?

What version of xplane2blender are you trying to install?

Which version of Blender are you using?

What version of OSX are you running?

-df

**Comment by ianm111** [comment]
Hi, thanks for replying1. New install of x2b so nothing to remove2. X2b is io_xplane2blender_4_2_0-rc_3-117_20230225221007-v12.zip3.  OSX 10.13.6However, I may have found the issue in that I just pulled the zip to my Mac rather than ‘dowload linked file’It seems that this automatically unzips which I gather is a bad thing.So I will remove x2b and start again from scratch with a new download and see if that solves it.Thanks again Ian Sent from my iPadOn 17 Jun 2024, at 03:49, DWmFrancis ***@***.***> wrote:﻿
Ianmm111



Did you remove the old version before attempting to install the new version of xplane2blender?



What version of xplane2blender are you trying to install?



Which version of Blender are you using?



What version of OSX are you running?



-df

**Comment by DWmFrancis** [comment]
That is correct. You should download the zip file, but don’t unzip it. Blender will install xplane2blender automatically, from the zip file.BTW, there is an earlier release of xplane2blender which can open as well as create XPlane object files. It is very educational.-dfOn Jun 17, 2024, at 03:48, ianm111 ***@***.***> wrote:﻿
Hi, thanks for replying1. New install of x2b so nothing to remove2. X2b is io_xplane2blender_4_2_0-rc_3-117_20230225221007-v12.zip3.  OSX 10.13.6However, I may have found the issue in that I just pulled the zip to my Mac rather than ‘dowload linked file’It seems that this automatically unzips which I gather is a bad thing.So I will remove x2b and start again from scratch with a new download and see if that solves it.Thanks again Ian Sent from my iPadOn 17 Jun 2024, at 03:49, DWmFrancis ***@***.***> wrote:﻿

Ianmm111







Did you remove the old version before attempting to install the new version of xplane2blender?







What version of xplane2blender are you trying to install?







Which version of Blender are you using?







What version of OSX are you running?







-df
